sql/sql_show.cc

```diff
@@ -2137,6 +2142,184 @@ class List_process_list : public Do_THD_Impl {
   }
 };
 
+typedef std::tuple<double, double, double> time_stat_tuple;
```

Comment:
Since it's "double, double, double", it is not clear which field means what, please consider a struct with three double fields instead